process/release_process.md

```diff
@@ -111,6 +111,31 @@ It leads us to need a separate 'xt-core' repository that will be the entry point
     - Merging the 'development' branch into the 'main' branch
     - Adding the commit that locks all the dependencies
     - Assigning the release tag to the HEAD commit of the 'main' branch
+- **Approach #3**:
+   - Development is done in the 'development' branch. In this branch, all dependencies can stick to the HEAD. In this stage we separate repositories that are including external dependencies and requires actual commit to stabilize external revision. They could be:
+     - Moulin meta repository which includes moulin.yaml
+     - Zephyr based repository with west.yaml which is the dependency for Moulin repository
+   - Release is done by creating a separate branch for each release on top-level repositories with the name of current release
+   - Version format as **<component>-vX.X.X** is acceptable because
```

Comment:
Hi @oleksiimoisieiev, it seems to be a sentence that has yet to be ended.  Acceptable because what? ))